prac_05/word_occurences.py

```diff
@@ -0,0 +1,21 @@
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual: 40 minutes
+"""
+
+# get user input and creat dictionary
```

Comment:
Spelling error in comment, though to be fair I would argue your code is clear enough that a comment isn't neccesary